src/RemoteInboxNotifications/RemoteInboxNotificationsEngine.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Handles running specs
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+namespace Automattic\WooCommerce\Admin\RemoteInboxNotifications;
+
+defined( 'ABSPATH' ) || exit;
+
+use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Note;
+use \Automattic\WooCommerce\Admin\PluginsProvider\PluginsProvider;
+
+/**
+ * RemoteInboxNotifications engine.
+ * This goes through the specs and runs (creates admin notes) for those
+ * specs that are able to be triggered.
+ */
+class RemoteInboxNotificationsEngine {
```

Comment:
Similar to other comments... I don't see an advantage of a class with all statics over namespaced functions.